src/sdkModels.ts

```diff
@@ -209,6 +211,10 @@ export class Parameter implements IParameter {
   asHashString() {
     return `${this.name}:${this.type.name}${this.required ? '' : '?'}${this.location}`
   }
+
+  doEncode() {
+    return this.type.name === 'string' || this.type.name === 'datetime' || this.type.name === 'date'
+  }
```

Comment:
👍

Reply:
Note that, because Swift and Kotlin method parameters are immutable, we can't easily take advantage of `param.doEncode()` until we clean up the code generator design a bit more.